### DISCUSSION TITLE
navbar order

### DISCUSSION THREAD

**Discussion by phpiando** [retypeapp/retype#231] [Q&A]
I am using retype to write a whitepaper for a project, however I would like to know how I can sort the navigation menu.

I've already tried adding the "order" position in my coin.md and economy.md files but it's not working.

How could I arrange the menu in the order I want?

![image](https://user-images.githubusercontent.com/12565400/144453276-7e3f3973-630e-4ef5-b720-7a883529ceb2.png)

**Comment by AndreyChechel** [comment]
Hi @over12,
thanks for interest in Retype!

Could you share configuration of the pages, or if your project is open-source - the GitHub repo URL, so we can review and suggest edits.

In general, the `order` config should help with sidebar item sorting, but there could be some exceptions. For instance, if a `SUMMARY.md` is used to form the sidebar content, the `order` configs will be ignored by Retype.

**Comment by geoffreymcgill** [comment]
I was able to configure the order using the following sample pages. 

<img width="282" alt="Screen Shot 2021-12-02 at 7 19 10 PM" src="https://user-images.githubusercontent.com/62210/144480407-1622a450-26d7-436a-827c-f62bb7655384.png">

#### economia.md

```md
---
order: 1000
---
# Economia

Sample Economia page.
```

#### coin.md

```md
# $COIN

Sample $COIN page
```

Hope this helps.

**Comment by phpiando** [comment]
Hi guys, thanks for the quick response.
I managed to solve it using the negative value in order.
